redaxo/src/core/lang/de_de.lang

```diff
@@ -230,6 +230,16 @@ sql_unable_to_create_database = Die angegebene Datenbank konnte nicht angelegt w
 sql_unable_to_find_database = Die angegebene Datenbank konnte nicht gefunden werden!
 sql_unable_to_open_database = Die angegebene Datenbank konnte nicht geöffnet werden!
 
+setup_security = Sicherheit
+
+https_activate_redirect_for = Automatische HTTPS-Umleitung
+https_disable = deaktiveren
+https_only_frontend = nur für Frontend aktivieren
+https_only_backend = nur für Backend aktivieren
+https_activate = für Frontend und Backend aktiveren
+
+hsts_activate = Setze HSTS Eintrag
```

Comment:
Ggf hier besser mit der ausformulierten Variante arbeiten, statt der Abkürzung

Reply:
Im besten Fall bräuchten wir dazu noch einen Eintrag in der Doku, damit man hier verlinken könnte

Reply:
>  https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security

siehe oben